components/molecules/Product/Product.js

```diff
@@ -0,0 +1,30 @@
+import {ProductStyled,TopSection,MiddleSection,BottomSection,Price,Buy,ProductDesc} from './style'
+import HeadingH4 from '../../atoms/Heading/HeadingH4'
+import Image from 'next/image'
+import TextP from '../../atoms/Text/TextP'
+import Button from '../../atoms/Button/Button'
+
+function Product({product}){
+    
+    return(
+        <ProductStyled>
+            <TopSection>
+                <HeadingH4>
+                    {product.name}
```

Comment:
Please use spread operator to get name,imageURL & other properties from product obj.